apps/rush/src/actions/UnlinkAction.ts

```diff
@@ -20,8 +20,9 @@ export default class UnlinkAction extends CommandLineAction {
     super({
       actionVerb: 'unlink',
       summary: 'Delete node_modules symlinks for all projects',
-      documentation: 'This removes the symlinks created by the "rush link" command,'
-        + ' allowing you to resume use NPM commands instead of Rush.'
+      documentation: 'This removes the symlinks created by the "rush link" command. This is useful for'
+       + ' cleaning a repo using "git clean" without accidentally deleting source files, or for using standard npm'
```

Comment:
The main scenarios I use "npm unlink" for are:

- If you want to run "npm install" without Rush

- It's faster to run "rush unlink" before trying to delete an entire repo folder, because Windows Explorer "recycle" processes symlinks very slowly

---
In reply to: [116345488](https://github.com/Microsoft/web-build-tools/pull/186#discussion_r116345488) [](ancestors = 116345488,116345448)